client/src/landing/AnonymousHome.js

```diff
@@ -0,0 +1,364 @@
+          the destination Renku accompanies the journey and helps you make sense of it.
+          All activity within Renku is captured in the Knowledge Graph.
+          This makes it possible to connect the dots, no matter where they lead.
+        </p>
+      </Col>
+    </Row>
+  </div>;
+}
+
+function TutorialLink(props) {
+  const url = props.url;
+  if ((url == null) || (url.length < 1))
+    return null;
+
+  if (url.startsWith("http")) {
+    return <ExternalLink
+      title="Follow the tutorial"
+      className="btn btn-outline-info" role="button" id="link-learn"
+      showLinkIcon={true}
+      url={url} />;
+  }
+  return <Link className="btn btn-outline-info" role="link" id="link-tutorial" to={url}>
+    Follow the tutorial
+  </Link>;
+}
+
+function Section4(props) {
+  const backgroundUrl = VisualDetail;
+  return <div id="rk-anon-home-section4"
+    style={{
+      backgroundImage: `url(${backgroundUrl})`
+    }}>
+    <div className="rk-w-s">
+      <div className="rk-pt-l">
+        <h3 className="text-info">Connecting dots</h3>
+      </div>
+      <div className="rk-pt-s">
+        <h3 className="text-white">The knowledge graph powers Renku and helps you make sense of what has been done.</h3>
+      </div>
+      <div className="rk-pt-s">
+        <h3 className="text-secondary">Ready to try it out?</h3>
+        <h3 className="text-info">Get started with Renku</h3>
+      </div>
+    </div>
+    <div className="d-flex flex-wrap">
+      <div className="pt-2" style={{ minWidth: "160px" }}>
+        <span>
+          <Link className="btn btn-outline-secondary me-1" role="button" id="link-sign_up" to="/login">
+        &nbsp;Sign Up
+          </Link>
+          (It&apos;s free)
+        </span>
+      </div>
+      <div className="pt-2" style={{ minWidth: "185px" }}>
+        <TutorialLink url={props.tutorialLink} />
+      </div>
+      <div className="pt-2" style={{ minWidth: "180px" }}>
+        <ExternalLink
+          title="Learn more"
+          className="btn btn-outline-info" role="button" id="link-learn"
+          showLinkIcon={true}
+          url="https://renku.readthedocs.io/en/latest/" />
+      </div>
+    </div>
+  </div>;
+}
+
+function Section5(props) {
+  return (props.projects == null) || (props.projects.length < 1) ?
+    <div id="rk-anon-home-section5-empty">
+      <div className="rk-pt-l">
+        <h3 className="text-info">&nbsp;</h3>
+      </div>
+    </div> :
+    <div id="rk-anon-home-section5">
+      <div className="rk-pt-l">
+        <h3 className="text-info">Look at some example projects</h3>
+      </div>
+    </div>;
+}
+
+function Section6(props) {
+  const backgroundUrl = VisualFooter;
+  return <div id="rk-anon-home-section6"
+    style={{
+      backgroundImage: `url(${backgroundUrl})`
+    }}>
+    <div>
+      <div><img src={logo} alt="Renku" height="68" className="d-block my-1" /></div>
+      <Row className="rk-pt-s" >
+        <Col xs={12} xl={4} >
```

Comment:
```suggestion
        <Col xs={12}>
```
Check comment at line 296